### DISCUSSION TITLE
%E5%89%8D%E8%A8%80/%E8%B4%A1%E7%8C%AE%E6%8C%87%E5%8D%97

### DISCUSSION THREAD

**Discussion by giscus** [SurviveXJTU/SurviveXJTU.github.io#2] [Announcements]
# %E5%89%8D%E8%A8%80/%E8%B4%A1%E7%8C%AE%E6%8C%87%E5%8D%97

西安交大生存指南，更适合西交大学子的生存指南。

https://survivexjtu.github.io/%E5%89%8D%E8%A8%80/%E8%B4%A1%E7%8C%AE%E6%8C%87%E5%8D%97.html

**Comment by Axi404** [comment]
欢迎大家按照贡献指南进行贡献，或者提出 Issue 等，相关疑问欢迎在下方留言！

**Reply by dousica** [reply]
# 你做的实在是太酷啦！😍
## 请问怎么联系你呢？

**Reply by Axi404** [reply]
可以通过邮件 axihelloworld@gmail.com，使用本评论区或者直接进行 PR 同样是一个好主意！

**Comment by jyhty** [comment]
很细致的教程，照着学习完全无压力，感谢分享！
